### PR TITLE
update hacking.md with oc commands and check for successful deployment.

### PR DESCRIPTION
**Pull request by kikisdeliveryservice**
Adding some of my notes to the mcd documentation.

### CODE REVIEW

**Comment by kikisdeliveryservice** [comment]
No idea why this docfix is failing these tests..

**Comment by ashcrow** [comment]
They are total flakes. No worries :smile:

File: HACKING.md [hunk 1]
```diff
@@ -22,12 +22,30 @@ These instructions were tested with installer version https://github.com/openshi
    CVO as it owns the configmap and it will revert your changes:
 
    ```sh
+   # if using kubectl:
    kubectl delete ds cluster-version-operator -n openshift-cluster-version
+
+   # if using oc:
+   oc delete daemonset cluster-version-operator -n openshift-cluster-version --config=auth/kubeconfig
```
Comment (jlebon) [inline HACKING.md]: Sidenote: I usually do `export KUBECONFIG=$(pwd)/auth/kubeconfig`, then you don't have to pass that switch every time.

Comment (kikisdeliveryservice) [reply HACKING.md]: thanks for the tip jlebon!

**Comment by abhinavdahiya** [comment]
Why is this oc and kubectl both, can't we just pick one? `oc` seems to be a super set of all the available features from kubectl, so just use `oc`?

i would rather add `export KUBECONFIG=<path to kubeconfig>` than add that `--config` flag to each command.

@ashcrow @kikisdeliveryservice

**Comment by ashcrow** [comment]
We can pick one. @kikisdeliveryservice's PR was meant to provide openshift commands. I'll follow up and remove the kubernetes commands.

**Pull request merged**